dev/Repeater/SelectionModel.cpp

```diff
@@ -619,32 +635,56 @@ void SelectionModel::SelectWithGroupImpl(int groupIndex, int itemIndex, bool sel
 void SelectionModel::SelectWithPathImpl(const winrt::IndexPath& index, bool select, bool raiseSelectionChanged)
 {
     bool selected = false;
+    bool selectionChanged = true;
```

Comment:
>selectionChanged [](start = 9, length = 16)

This variable seems overloaded, you use it to get into the next block in the single select case and then use it for a different utility after that. Instead use two different variables please.

Reply:
Added a new variable for that.